dotcom-rendering/src/palette.ts

```diff
@@ -71,6 +71,14 @@ const headlineBackgroundColourDark = ({ design }: ArticleFormat): string => {
 	}
 };
 
+const headlineBorder = ({ design }: ArticleFormat): string => {
+	if (design === ArticleDesign.LiveBlog) {
+		return 'rgba(255,255,255, 0.2)';
+	}
+	if (design === ArticleDesign.DeadBlog) return '#CDCDCD';
```

Comment:
non-blocking

I wonder if '#CDCDCD' - which isn't in source - is a mistake for neutral[86] ('#DCDCDC').  Maybe @HarryFischer or @benwuersching might know?